bricks/dart_frog_new/hooks/analysis_options.yaml

```diff
@@ -0,0 +1,5 @@
+include: package:very_good_analysis/analysis_options.4.0.0.yaml
+
+linter:
+  rules:
+    public_member_api_docs: false
```

Comment:
Can we enable this 🥺 ?

Reply:
No.

Reply:
We tend to deactivate this rule on packages with an application character (rather than a library), see [here](https://github.com/VeryGoodOpenSource/very_good_core/blob/main/src/my_app/analysis_options.yaml#L4) and [here](https://github.com/VeryGoodOpenSource/very_good_dart_cli/blob/main/src/my_cli/analysis_options.yaml#L4).